### ISSUE TITLE
VS extensibility APIs required to unblock net5 TFMs with platform identifiers

### ISSUE THREAD

**Issue by zivkan** [NuGet/Home#9748]
Investigate existing VS extensibility APIs related to framework parsing, framework names, etc, to understand how other (official) VS components use NuGet and how they'll be impacted for net5.0 TFMs with a platform identifier.

Once this is understood, design and implement new APIs to unblock usage of TFMs with TPI's in VS.

**Comment by zivkan** [comment]
duplicate of https://github.com/NuGet/Home/issues/9650
